### PR TITLE
resource/alicloud_ess_scaling_configuration: add burstable_performance excluded_instance_types and architectures attributes of instance_pattern_info

### PR DESCRIPTION
**Pull request by fuliu-zln**
…e excluded_instance_types and architectures attributes of instance_pattern_info

### CODE REVIEW

File: alicloud/resource_alicloud_ess_scaling_configuration.go [hunk 1]
```diff
@@ -352,6 +352,25 @@ func resourceAlicloudEssScalingConfiguration() *schema.Resource {
 							Optional:     true,
 							ValidateFunc: validation.StringInSlice([]string{"EntryLevel", "EnterpriseLevel", "CreditEntryLevel"}, false),
 						},
+						"burstable_performance": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Default:      "Include",
+							ValidateFunc: validation.StringInSlice([]string{"Exclude", "Include", "Required"}, false),
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_ess_scaling_configuration.go:359]: validation去掉，只要 StringInxxx 这个就行

Comment (fuliu-zln) [reply alicloud/resource_alicloud_ess_scaling_configuration.go]: done

File: alicloud/resource_alicloud_ess_scaling_configuration.go [hunk 2]
```diff
@@ -350,7 +348,26 @@ func resourceAlicloudEssScalingConfiguration() *schema.Resource {
 						"instance_family_level": {
 							Type:         schema.TypeString,
 							Optional:     true,
-							ValidateFunc: validation.StringInSlice([]string{"EntryLevel", "EnterpriseLevel", "CreditEntryLevel"}, false),
+							ValidateFunc: StringInSlice([]string{"EntryLevel", "EnterpriseLevel", "CreditEntryLevel"}, false),
+						},
+						"burstable_performance": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Default:      "Include",
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_ess_scaling_configuration.go:356]: 这个Default去掉，改成Computed

Comment (fuliu-zln) [reply alicloud/resource_alicloud_ess_scaling_configuration.go]: done

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**